src/main/java/convobot/commands/Delete.java

```diff
@@ -0,0 +1,44 @@
+package commands;
+
+import exceptions.ConvoBotException;
+import utils.ResponseConstructor;
+import utils.TaskList;
+
+/**
+ * Represents a command to delete a task from the task list.
+ */
+public class Delete implements Command {
+
+    private final int i;
```

Comment:
Maybe can make the variable name more meaningful like index because the scope is not small. Same issue in different places too.

Reply:
Accepted, renamed to indexToDelete.